website/build.gradle

```diff
@@ -269,9 +269,10 @@ task commitWebsite {
 
     def git = grgit.open(currentDir: project.rootDir)
     // get the latest commit on master
-    def latestCommit = grgit.log(maxCommits: 1)[0].abbreviatedId
```

Comment:
What's the difference between abbreviatedId and id. I couldn't find much in documentation.

Reply:
The abbreviatedId is the short sha-1 hash of a commit, and id is the full 40-char sha1 hash. I don't think this change helped, so I reverted the change.